bin/node-template/runtime/src/template.rs

```diff
@@ -29,17 +29,21 @@ decl_storage! {
 	}
 }
 
-// The module's dispatchable functions.
+// The pallet's dispatchable functions.
 decl_module! {
 	/// The module declaration.
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+		// Initializing errors
+		// this is needed only if you are using errors in your pallet
```

Comment:
Maybe also say that you need this to expose the error in the Metadata.

Reply:
You mean in `construct_runtime!`? I just realized I didn't actually do that in the node template. I'll add it there too.

Reply:
What @bkchr is saying is that adding `type Error = Error<T>` also tells `decl_module` to generate the right stuff to expose the Error enum in your metadata. No need to change code, simply update the comment to include that this line also exposes information in the metadata.